core-api/src/main/java/com/optimizely/ab/Optimizely.java

```diff
@@ -160,8 +161,8 @@ private Optimizely(@Nonnull ProjectConfig projectConfig,
         LogEvent impressionEvent =
                 eventBuilder.createImpressionEvent(projectConfig, experiment, variation, userId, attributes);
         logger.info("Activating user \"{}\" in experiment \"{}\".", userId, experiment.getKey());
-        logger.debug("Dispatching impression event to URL {} with params {}.", impressionEvent.getEndpointUrl(),
-                     impressionEvent.getRequestParams());
+        logger.debug("Dispatching impression event to URL {} with params {} and payload \"{}\".",
+                     impressionEvent.getEndpointUrl(), impressionEvent.getRequestParams(), impressionEvent.getBody());
```

Comment:
How is adding payload helping? Wouldn't there be a lot of overlap between params and body already?